modules/machine-config-node-disruption-config.adoc

```diff
@@ -0,0 +1,135 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="machine-config-node-disruption-config_{context}"]
+= Configuring node restart behaviors after machine config changes
+
+You can create a node disruption policy to define the machine configuration changes that cause a disruption to your cluster, and which changes do not.
+
+You can control how your nodes respond to changes in the configuration files in the `/var` or `/etc` directory, systemd units, or SSH keys.
+
+include::snippets/machine-config-node-disruption-actions.adoc[]
+
+.Prerequisite
+
+
+* You have enabled the `TechPreviewNoUpgrade` feature set by using the feature gates. For more information, see _Enabling features using feature gates_.
++
+[WARNING]
+====
+Enabling the `TechPreviewNoUpgrade` feature set on your cluster prevents minor version updates. The `TechPreviewNoUpgrade` feature set cannot be disabled. Do not enable this feature set on production clusters.
```

Comment:
@mburke5678 what does the minor version mean here, y or z? as I know z-stream level upgrade is allowed.

Reply:
@rioliu-rh I don't know, to be honest. This is standard boilerplate wording that we use. I believe it means y-stream updates.

Reply:
you're right, the version pattern is like major.minor.patch
xref: https://www.redhat.com/en/blog/the-ultimate-guide-to-openshift-update-for-cluster-administrators